ChangeLog.md

```diff
@@ -18,7 +18,9 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
-2.0.28
+2.0.29 - (not yet released)
```

Comment:
We probably don't want to have `(not yet released)` otherwise it prevents arbitrary versions from bring release candidates.

Reply:
Hm, good point.